assets/schema_input.json

```
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/ebi-metagenomics/mobilome-annotation-pipeline/master/assets/schema_input.json",
+    "title": "ebi-metagenomics/mobilome-annotation-pipeline pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "sample": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
+            },
+            "assembly": {
+                "type": "string",
+                "pattern": "^\\S+\\.(fa|fna|fasta)$",
```

```suggestion
                "pattern": "^\\S+\\.(fa|fna|fasta)(\\.gz)?$",
```
Switch to this, if the pipeline can handle `.gz` assembly inputs.

---

Only uncompressed fasta files are supported at the moment. I can see it is the same for the [mettanotator](https://github.com/EBI-Metagenomics/mettannotator/blob/a70e121334cf4ed14957a133d58dc806bcdac1c7/assets/schema_input.json#L16-L19). I will keep it as it is for now.